res/database/Default/database_ads.xml

```diff
@@ -1314,7 +1316,7 @@ Brucey - harsh fragrance for male experts.</en>
 			</title>
 			<description>
 				<de>Bei uns wird Ihr Buch noch gewalzt! Egal ob Stammbuch oder Tagebuch, wir bringen Ihr Werk unters Volk.</de>
-				<en>We still mill your book! Whether register or diary, we propagate it to the hoi poloi.</en>
+				<en>We still mill your book! Whether family book or diary, we get your work to the masses.</en>
```

Comment:
Although "hoi poloi" was misspelled (it is "hoi polloi") I think the term is rather well fitting to the "sophisticated" people of the book producers :)
Yet you are right: many people won't know the term, so "masses" is the better option.

Reply:
it's also just closer to the original German. `:)`